frontend/src/features/mapSideBar/SidebarContents/LandForm.tsx

```diff
@@ -190,20 +188,6 @@ const Form: React.FC<ILandForm> = ({
         return (
           <LandValuationForm title="Bare Land Valuation" nameSpace="data" disabled={disabled} />
         );
-      case LandSteps.ASSOCIATED_OR_REVIEW:
```

Comment:
Alex, can you confirm that both bare and developed land work properly with these changes?

Reply:
To answer your earlier comment: the steps on adding associated land show as land Ownership - Identification - Usage - Valuation - review & Submit, and on the final review page it doesn't show any buildings, which is the same as what's in dev right now

Reply:
Just checking the others now

Reply:
As far as I can tell, they both work properly